.travis.yml

```diff
@@ -2,19 +2,20 @@ sudo: false
 cache: bundler
 language: ruby
 services: memcache
-before_install:
-  - gem update bundler
+
 rvm:
-  - 2.2.9
-  - 2.3.6
-  - 2.4.3
-  - 2.5.0
+  - "2.5"
+  - "2.6"
+  - "2.7"
+
 env:
-  - "AS_VERSION=4.0.5"
-  - "AS_VERSION=4.1.0"
-  - "AS_VERSION=4.2.0"
-  - "AS_VERSION=5.0.0"
-  - "AS_VERSION=5.1.0"
-  - "AS_VERSION=5.2.0.rc1"
+  - "AS_VERSION=5.2.0"
+  - "AS_VERSION=6.0.0"
```

Comment:
Should that be `AS_VERSION=~6.0.0`  instead?

Reply:
Why? That env var is interpolated here: https://github.com/Shopify/memcached_store/blob/0b2b0594ce0b5c669e076565cbc26c9c62d5d04f/Gemfile#L11